src/transformState.js

```diff
@@ -4,8 +4,25 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
You can also make destructuring to get all needed props of action to make your code cleaner